go.mod1

```diff
@@ -55,3 +55,5 @@ require (
 )
 
 replace cloud.google.com/go/storage => github.com/3pointer/google-cloud-go/storage v1.6.1-0.20210108125931-b59bfa0720b2
+
+replace github.com/pingcap/kvproto => github.com/Little-Wallace/kvproto v0.0.0-20210531082236-c0218d95203e
```

Comment:
pingcap/kvproto#762 has been merged, please change the `replace` into a proper dependency